### PR TITLE
gh-251 Fix documentation bugs

### LINKED ISSUE #251: Fix documentation bugs
When reading through the docs I noticed some minor bugs:

- Missing code block (Operation Chains)
- Broken links (Traversing)
- Java main function missing (Traversing)

### PR DESCRIPTION
**Pull request by GCHQDeveloper314**
# Related Issue

- Resolve #251

### CODE REVIEW

File: src/main/resources/user/walkthrough/Traversing.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Here are the results:
 ${GET_ADJACENT_IDS_SIMPLE_RESULT}
 ```
 
-For more details and for more detailed examples of the GetAdjacentIds operation please see [GetAdjacentIds](operations/getadjacentids.md)
+For more details and for more detailed examples of the GetAdjacentIds operation please see [GetAdjacentIds](../operations/getadjacentids.md)
```
Comment (t92549) [inline src/main/resources/user/walkthrough/Traversing.md:52]: Does `..` work once converted to HTML?

File: src/main/java/uk/gov/gchq/gaffer/doc/user/walkthrough/Traversing.java [hunk 2]
```diff
@@ -179,4 +179,9 @@ public Traversing() {
         }
         return printableWalks;
     }
+
+    public static void main(final String[] args) throws OperationException, IOException {
+        final Traversing walkthrough = new Traversing();
+        walkthrough.run();
```
Comment (t92549) [inline src/main/java/uk/gov/gchq/gaffer/doc/user/walkthrough/Traversing.java:185]: What was missing from this?

File: src/main/resources/user/walkthrough/Traversing.md (hunk 1 above)
Comment (GCHQDeveloper314) [reply src/main/resources/user/walkthrough/Traversing.md:52]: I checked and it does work. It get converted to a full (non-relative) HTML path to the page. In HTML I think using `..` may or may not work depending on the server, which probably explains why it's done this way.
This type of path is also used in Intro.md, TheBasics.md, Filtering.md and other pages.

File: src/main/java/uk/gov/gchq/gaffer/doc/user/walkthrough/Traversing.java (hunk 2 above)
Comment (GCHQDeveloper314) [reply src/main/java/uk/gov/gchq/gaffer/doc/user/walkthrough/Traversing.java:185]: All the other walkthrough code classes have a main function, but for some reason this one didn't which prevented it from being run manually. It still worked for the build itself as that creates an object of this class elsewhere and doesn't call main.

**Review by t92549 — APPROVED** [review]

**Pull request merged**